contrib/exporters/secadvisor/mod/extend_gremlin.go

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"text/template"
+	"time"
+
+	cache "github.com/pmylund/go-cache"
+	"github.com/skydive-project/skydive/api/client"
+	"github.com/skydive-project/skydive/contrib/exporters/core"
+	"github.com/skydive-project/skydive/logging"
+	"github.com/spf13/viper"
+)
+
+// extendGremlin is used for transform extensions using gremlin expressions
```

Comment:
I don't think you need such a big comment - what is obvious from the code does not require explanations